api/prisma/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
+provider = "sqlite"
```

Comment:
This might be an issue to commit

Reply:
Actually I think this is just so that it will only run the migration if the provider matches